xarray/core/dask_array_ops.py

```diff
@@ -24,3 +26,44 @@ def dask_rolling_wrapper(moving_func, a, window, min_count=None, axis=-1):
     # trim array
     result = da.ghost.trim_internal(out, depth)
     return result
+
+
+def rolling_window(a, window, axis=-1):
+    """ Dask's equivalence to np.utils.rolling_window """
+    # inputs for ghost
+    if axis < 0:
+        axis = a.ndim + axis
+    depth = {d: 0 for d in range(a.ndim)}
+    if window % 2 == 0:
+        depth[axis] = int((window - 1) / 2 + 1)
+        offset = 1
+    else:
+        depth[axis] = int((window - 1) / 2)
+        offset = 0
+
+    if depth[axis] > min(a.chunks[axis]):
+        raise ValueError(
+            "The window size %d is larger than your\n"
+            "smallest chunk size %d + 1. Rechunk your array\n"
+            "with a larger chunk size or a chunk size that\n"
+            "more evenly divides the shape of your array." %
+            (window, min(a.chunks[axis])))
+
+    boundary = {d: np.nan for d in range(a.ndim)}
```

Comment:
Should use this `fill_value` passed on from the Variable method instead of always being `np.nan`?